### PR TITLE
fix predict result col value

### LINKED ISSUE #1348: predict sql
**Is your feature request related to a problem? Please describe.**
A clear description of the problem

**Describe the solution you'd like**
When I run the predict sql `select * from sqlflow_test_set TO PREDICT iris_dnn_predict233.class2 using iris_dnn_model33;`
the data insert to the col class(label column）,not the target column class2

**Additional Notes**
The test table sqlflow_test_set
image
The predict table iris_dnn_predict233
image

### PR DESCRIPTION
**Pull request by Yancey0623**
fixed #1348

### CODE REVIEW

File: pkg/sql/executor_ir.go [hunk 1]
```diff
@@ -307,6 +307,7 @@ func createPredictionTable(predParsed *extendedSelect, db *DB, session *pb.Sessi
 // Create prediction table using the `PredictStmt`.
 // TODO(typhoonzero): remove legacy `createPredictionTable` once we change all submitters to use IR.
 func createPredictionTableFromIR(predStmt *ir.PredictStmt, db *DB, session *pb.Session) error {
+	fmt.Println("createPredictionTableFromIR")
```
Comment (typhoonzero) [inline pkg/sql/executor_ir.go]: remove debug code?

File: pkg/sql/codegen/tensorflow/codegen.go [hunk 2]
```diff
@@ -312,8 +312,11 @@ func Pred(predStmt *ir.PredictStmt, session *pb.Session) (string, error) {
 			fmt.Sprintf("\"%s\": [%s]", target, strings.Join(perTargetFeatureColumnsCode, ",\n")))
 	}
 	isKeras, estimatorStr := IsKerasModel(predStmt.TrainStmt.Estimator)
-	labelFM := predStmt.TrainStmt.Label.GetFieldMeta()[0]
-	if labelFM.Name == "" {
+	var labelFM *ir.FieldMeta
+	if predStmt.TrainStmt.Label != nil {
+		labelFM = predStmt.TrainStmt.Label.GetFieldMeta()[0]
+		labelFM.Name = predStmt.ResultColumn
```
Comment (typhoonzero) [inline pkg/sql/codegen/tensorflow/codegen.go]: Seems this PR only keeps one label column that specified in the predict SQL?

Comment (Yancey0623) [reply pkg/sql/codegen/tensorflow/codegen.go]: Yes, always using `predict result column` as the prediction result column name.

**Review by typhoonzero — APPROVED** [review]
LGTM

**Pull request merged**